monitoring/nethealth.go

```diff
@@ -59,9 +62,13 @@ func (c *NethealthConfig) CheckAndSetDefaults() error {
 	if c.AdvertiseIP == "" {
 		errors = append(errors, trace.BadParameter("host advertise ip must be provided"))
 	}
+	if c.SerfClient == nil {
+		errors = append(errors, trace.BadParameter("local serf client can't be empty"))
+	}
 	if c.KubeConfig == nil {
 		errors = append(errors, trace.BadParameter("kubernetes access config must be provided"))
 	}
+
```

Comment:
Nit: Bit inconsistent to have an empty line here compared to the rest of this method.